### ISSUE TITLE
Classes: examples don't show up unless a description is present?

### ISSUE THREAD

**Issue by ambischof** [jsdoc2md/jsdoc-to-markdown#73]
Probably because the example is put in the same section as the description, but if the description is not set, the example doesn't show up.

**Comment by 75lb** [comment]
can you post some code i can use to reproduce, please

**Comment by ambischof** [comment]
```
/**
 * @class Storage
 * @classdesc
 * An app wide storage/state persistence solution with direct and radio connectivity.
 * Can be accessed:
 * * directly through the instance via member functions,
 * * through the radio channel "storage"
 *
 * Typically it's probably better to access through the radio, so that there's no need to have a reference to app.
 *
 * **Note: There can only be one instance of this class.** Because there can only be one instance of the "storage" radio channel.
 *
 * Radio Handles:
 * * "set:item" -> radio proxy for setItem (same parameters)
 * * "get:item" -> radio proxy for getItem (same parameters and return)
 *
 * @todo Use localStorage or sessionStorage
 * @todo Further consider the identifier scheme for pages. Right now pages use their url, but it may not be the best way?
 * @example
 * app.storage = new Storage();
 *
 * app.storage.setItem('userdashboard', 'foo', 'bar');
 * app.storage.getItem('userdashboard', foo);
 * // -> "bar"
 *
 * //same result as above.
 * Backbone.Radio.channel('storage').trigger('set:item', 'userdashboard', 'foo', 'bar');
 * Backbone.Radio.channel('storage').request('get:item', 'userdashboard', 'foo');
 * // -> "bar"
 */
```

If you add a line or two at the top for the description, the example is rendered as expected.

**Comment by 75lb** [comment]
ok, i see it now.. thanks - will look in the morning..

**Comment by mkhazov** [comment]
In my case @example doesn't show up even if the description is set.

**Comment by danielyaa5** [comment]
I think this has to do with transforming for es-2015 and es-2017 features

**Comment by danielyaa5** [comment]
You need @method see this https://github.com/ctumolosus/jsdoc-babel/issues/1

**Comment by ambischof** [comment]
@danielyaa5
I don't think that's relevant for this issue. 
Besides, @method is short for @type method and @class is short for @type class. It wouldn't make sense to use that.

**Comment by danielyaa5** [comment]
@ambischof  
Have you tried this? It got it working for me, so it seems relevant

**Comment by 75lb** [comment]
Could you check whether this issue is resolved by [jsdoc2md v5.0.0](https://github.com/jsdoc2md/jsdoc-to-markdown/releases/tag/v5.0.0) please?

**Comment by 75lb** [comment]
A PR to address this issue was merged and released, let me know if there's anything else. Feel free to also test the [jsdocm2d v9 pre-release](https://github.com/jsdoc2md/jsdoc-to-markdown/releases/) if you like.
